ecs-init/docker/docker.go

```diff
@@ -243,11 +245,28 @@ func (c *Client) getContainerConfig() *godocker.Config {
 	for envKey, envValue := range envVariables {
 		env = append(env, envKey+"="+envValue)
 	}
-
-	return &godocker.Config{
+	cfg := &godocker.Config{
 		Env:   env,
 		Image: config.AgentImageName,
 	}
+	setLabels(cfg, envVariables["ECS_AGENT_LABELS"])
+	return cfg
+}
+
+func setLabels(cfg *godocker.Config, lablesStringRaw string) {
```

Comment:
a few typos in this function: lables -> labels

Reply:
:) I get caught with this all the time. UK based people want it different... Anyway its been fixed in the next commit.